app/controllers/api/v1/hosts_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::V1::HostsController < ApplicationController
+
+  # /api/v1/hosts
+  def index
+    # TODO this is a stopgap. It returns exactly what the UI wants
+    # so we don't have to build associations and make multiple
+    # requests for the /admin/overview/hosts page.
+    # When the client side becomes a little more mature, we should
+    # be make this endpoint a little less terrible and only serve host info
+    host_json = User.hosts.map do |user|
+      {
+        name:                user.name,
+        number_of_teatimes:  user.tea_times.count,
+        last_teatime:        user.tea_times.where('start_time <= ?', Time.now.utc).last,
+        next_teatime:        user.tea_times.where('start_time <= ?', Time.now.utc).last,
+        total_attendees:     user.tea_times.map {|t| t.attendances.count}.inject(0, &:+),
```

Comment:
Probably a question for @ankitshah811. I just used the same code that was in the Rails view.